Sources/PlayerView.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  PlayerView.swift
+//  PlayerKit
+//
+//  Created by Balatbat, Bryant on 12/20/17.
+//
+
+/// This file creates a `typealias` called `PlayerView` that determines which core view component to use depending on
+/// which platform the binary is being compiled for. If iOS or tvOS, `UIKit` will be imported and `UIView` will be used.
+/// otherwise, if macOS is used, `AppKit` will be imported and `NSView` will be used.
+
+#if os(iOS) || os(tvOS)
+    import UIKit
+#else
+    import AppKit
+#endif
+
+#if os(iOS) || os(tvOS)
+    public typealias PlayerView = UIView
```

Comment:
Could we use `canImport(AppKit)` and `canImport(UIKit)` here instead?

Reply:
https://www.hackingwithswift.com/example-code/language/how-to-check-whether-a-module-is-available-using-canimport

Reply:
Also, maybe 

```
#if canImport(UIKit)
    // ...
#elseif canImport(AppKit)
    // ...
#else 
    public typealias PlayerView = Never
```
?

Reply:
Okay, that seems more succinct and contextually appropriate.